pom.xml

```diff
@@ -2136,7 +2136,7 @@
         <carbon.deployment.version>4.10.2</carbon.deployment.version>
         <carbon.commons.version>4.7.28</carbon.commons.version>
         <carbon.registry.version>4.7.33</carbon.registry.version>
-        <carbon.multitenancy.version>4.9.0</carbon.multitenancy.version>
+        <carbon.multitenancy.version>4.9.2</carbon.multitenancy.version>
```

Comment:
@IsuraD Please verify.

Reply:
Have we done the changes in identity.xml ?

Reply:
Yes. Those changes have been done in this PR[1]

[1] https://github.com/wso2/carbon-identity-framework/pull/3084/files